Packages/com.unity.ide.rider/Rider/Editor/ProjectGeneration.cs

```diff
@@ -820,6 +826,58 @@ string SolutionText(IEnumerable<Assembly> islands)
         relevantIslands.Select(i => GetProjectActiveConfigurations(ProjectGuid(i.name))).ToArray());
       return string.Format(GetSolutionText(), fileversion, vsversion, projectEntries, projectConfigurations);
     }
+    
+    private static string GenerateAnalyserItemGroup(List<ResponseFileData> responseFilesData)
+    {
+        var paths = GetValuesFromResponseFilesData(responseFilesData, "analyzer")
+          .Concat(GetValuesFromResponseFilesData(responseFilesData, "a")).Distinct().ToArray();
+
+        //   <ItemGroup>
+        //      <Analyzer Include="..\packages\Comments_analyser.1.0.6626.21356\analyzers\dotnet\cs\Comments_analyser.dll" />
+        //      <Analyzer Include="..\packages\UnityEngineAnalyzer.1.0.0.0\analyzers\dotnet\cs\UnityEngineAnalyzer.dll" />
+        //  </ItemGroup>
+        if (!paths.Any())
+            return string.Empty;
+
+        var list = new List<string> {"<ItemGroup>"};
+        list.AddRange(paths.Select(a => $"    <Analyzer Include=\"{a}\" />"));
+        list.Add("  </ItemGroup>");
+        return string.Join("\r\n", list.ToArray());
+    }
+
+    private static string[] GetValuesFromResponseFilesData(List<ResponseFileData> responseFilesData, string key)
+    {
+      var paths = responseFilesData.SelectMany(x =>
+      {
+        string start = $"/{key}:";
```

Comment:
Keep in mind that the arguments do not have to start with a `/`; ` `-` is valid as well

Reply:
Nice catch, thanks